### PR TITLE
Misuv 6445 The back button for a individual user points to agent url from View and Manage Income Sources

### PR DESCRIPTION
**Pull request by tharunprasath**

### CODE REVIEW

File: app/controllers/incomeSources/manage/ManageIncomeSourceController.scala [hunk 1]
```diff
@@ -57,8 +57,8 @@ class ManageIncomeSourceController @Inject()(val manageIncomeSources: ManageInco
       sources = user.incomeSources,
       isAgent = isAgent,
       backUrl = {
-        if(isAgent) controllers.routes.HomeController.show()
-        else controllers.routes.HomeController.showAgent
+        if(isAgent) controllers.routes.HomeController.showAgent
```
Comment (avelx) [inline app/controllers/incomeSources/manage/ManageIncomeSourceController.scala:60]: any tests associated with this change ?

Comment (tharunprasath) [reply app/controllers/incomeSources/manage/ManageIncomeSourceController.scala:60]: We haven't written a test case for the backlink as part of UT or IT. But this should be covered according to the AT journey coverage. Do we still need to write UT for this fix?

Comment (avelx) [reply app/controllers/incomeSources/manage/ManageIncomeSourceController.scala:60]: ATs not failing for this change, so I would think that this change is not covered on any level of tests

**Review by avelx — APPROVED** [review]
approved

**Pull request merged**